adapters/coinzilla/coinzilla_test.go

```diff
@@ -0,0 +1,23 @@
+package coinzilla
+
+import (
+	"testing"
+
+	"github.com/prebid/prebid-server/adapters/adapterstest"
+	"github.com/prebid/prebid-server/config"
+	"github.com/prebid/prebid-server/openrtb_ext"
+)
+
+const testsDir = "coinzillatest"
+const testsBidderEndpoint = "http://request-global.czilladx.com/serve/prebid-server.php"
```

Comment:
Just FYI: You don't need to provide a real endpoint for tests here. You can provide some mock test endpoint. This is only used for assertions on the http calls specified in the JSON tests.

Reply:
Updated.Thanks!